package.json

```diff
@@ -23,7 +23,8 @@
     "release": "./scripts/release.sh"
   },
   "prettier": {
-    "singleQuote": true
+    "singleQuote": true,
+    "trailingComma": "none"
```

Comment:
I don't really care that much, but this keeps the existing behavior that was present before the upgrade to keep things consistent